opencti-platform/opencti-graphql/src/manager/activityListener.ts

```diff
@@ -197,6 +197,21 @@ const initActivityManager = () => {
           const message = `asks for \`${entity_name}\` enrichment with connector \`${connector_name}\``;
           await activityLogger(action, message);
         }
+        if (action.event_scope === 'enrich') {
+          const { entity_name, connector_name } = action.context_data;
+          const message = `asks for \`${entity_name}\` enrichment with connector \`${connector_name}\``;
+          await activityLogger(action, message);
+        }
+        if (action.event_scope === 'enrich') {
```

Comment:
looks like there was an issue here, `if (action.event_scope === 'enrich') {` block is present 3 times, I guess we only need `if (action.event_scope === 'analyze') {` block